File: apps/institution-api/Models/InstitutionEmail.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionEmail
    {
        public int InstitutionEmailId { get; set; }
        public int InstitutionId { get; set; }
        public Guid? EmailGuid { get; set; }
        public string EmailUsage { get; set; }
        public string EmailType { get; set; }
        public int? ListSequence { get; set; }
        public string EmailAddress { get; set; }
        public DateTime? EmailBusinessStartDate { get; set; }
        public DateTime? EmailBusinessEndDate { get; set; }

        public virtual Institution Institution { get; set; }
    }
}

File: libs/common/models/institution-email.ts


import { Institution } from './institution';

export class InstitutionEmail {
    public institutionEmailId: number = 0;
    public institutionId: number = 0;
    public emailGuid: string;
    public emailUsage: string = '';
    public emailType: string = '';
    public listSequence: number;
    public emailAddress: string = '';
    public emailBusinessStartDate: Date;
    public emailBusinessEndDate: Date;
    public institution: Institution = new Institution();

    constructor(init? : Partial<InstitutionEmail>) {
        
        Object.assign(this, init);
    }
}

